Automattic/wp-calypso
# Domains step - use your domain: Remove subheader text

Related to https://github.com/Automattic/dotcom-forge/issues/8750

## Proposed Changes

* Remove the "Find and claim one or more domain names" subheader text from the use your domain page
 
## Why are these changes being made?


* UI improvement

## Testing Instructions



* Use the live link and go to /start/domains
* Click on the `Use a domain I own` link in the sidebar (right sidebar)
* Notice that we don't display that subheader text
* Fill in a domain and click Continue
* Notice that the subheader is still not visible.

## Pre-merge Checklist
